src/UI/class-settings-page.php

```diff
@@ -638,12 +638,19 @@ public function validate_options( array $input ): array {
 		$input['track_page_types'] = self::validate_option_array( $input['track_page_types'] );
 
 		// Detect and prevent duplicate tracking.
-		$duplicate_tracking_found = ! empty( array_intersect( $input['track_post_types'], $input['track_page_types'] ) );
-		if ( $duplicate_tracking_found ) {
+		$duplicate_items = array_intersect( $input['track_post_types'], $input['track_page_types'] );
+		if ( ! empty( $duplicate_items ) ) {
```

Comment:
Instead of the generic `empty()`, can we use a more specific `0 === count( $duplicate_items )`?

https://beberlei.de/2021/02/19/when_to_use_empty_in_php_i_say_never.html

Reply:
I'm certainly not against it.

Looking at the current codebase of the plugin, we use `empty()` in a variety of places, which brings up the question of whether a refactor should be done at some point. In general I would say that I `empty()` is more readable for many people as well as shorter, but we could go with the alternatives if we want to aim on better quality code.

For the time being, I'll post a commit addressing this.